sdk/textanalytics/Azure.AI.TextAnalytics/tests/DetectLanguageTests.cs

```diff
@@ -266,11 +266,27 @@ public void DetectLanguageBatchDisableServiceLogsThrows()
             Assert.AreEqual("TextAnalyticsRequestOptions.DisableServiceLogs is not available in API version v3.0. Use service API version v3.1 or newer.", ex.Message);
         }
 
+        [RecordedTest]
+        [ServiceVersion(Min = TextAnalyticsClientOptions.ServiceVersion.V2022_10_01_Preview)]
+        public async Task DetectLanguageBatchWithScriptTest()
+        {
+            string document = "Tumhara naam kya hai?";
+
+            TextAnalyticsClient client = GetClient();
+            TextAnalyticsRequestOptions options = new TextAnalyticsRequestOptions() { ModelVersion = "2022-04-10-preview" };
```

Comment:
I'd add a `BUGBUG` comment to this tracking when this should be changed. Ideally we don't have to pin to such an old - even pre-GA (2022-05-01) - version.

Reply:
Could you remind me why we need to pin the model version? Is it to make sure the behavior of the test won't change?

Reply:
@heaths: Good call. I created an issue and added a comment here.

@kinelski: The reason in this case is that currently model version 2022-04-10-preview is the only one that supports this feature, which is kind of unfortunate and a little awkward.